examples/text_labels_and_annotations/custom_legends.py

```diff
@@ -33,10 +33,10 @@
 
 fig, ax = plt.subplots()
 lines = ax.plot(data)
-ax.legend(lines)
```

Comment:
The check caught this wrong usage in an example.

Fixed to do what is claimed to be done above (" If you simply plot the lines
want a legend item to show up for each one. If you simply plot the lines
and call ``ax.legend()``, you will get the following:").

Overall the example is only semi-instructive. I will keep a rewrite for a separate PR. I just made the minimal change here to make it correct and consistent.

Reply:
I wouldn't call it wrong per se. If you remove the argument, no legend will be there at all - which is not what the example wants to convey. It should also trigger a warning like "No handles with labels found" or similar?

Reply:
Maybe not technically wrong, but carelessly written and misleading. See my comment below.